core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/worked-example-editor.directive.html

```diff
@@ -35,17 +36,74 @@
       <div class="float-right">
         <button type="button"
                 class="btn btn-secondary"
-                ng-click="cancelEdit()">
+                ng-click="cancelEditQuestion()">
           Cancel
         </button>
         <button type="button"
                 class="btn btn-success"
-                ng-disabled="!container.workedExampleHtml || editWorkedExampleForm.$invalid"
-                ng-click="saveWorkedExample()">
+                ng-disabled="!container.workedExampleQuestionHtml || editWorkedExampleForm.$invalid"
+                ng-click="saveWorkedExample(true)">
+          Save
+        </button>
+      </div>
+      <div style="clear: both;"></div>
+    </div>
+  </div>
+
+  <!-- Answer Editor -->
```

Comment:
Hm. Maybe we should just change it to "Explanation" throughout. I think the only place where we need to see the word "Answer" is in the learner/creator UI, but all the code (backend/frontend) should be talking about explanations.

Reply:
Done